src/node_options.cc

```diff
@@ -351,11 +351,6 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             &EnvironmentOptions::enable_source_maps,
             kAllowedInEnvvar);
   AddOption("--experimental-abortcontroller", "", NoOp{}, kAllowedInEnvvar);
-  AddOption("--experimental-fetch",
```

Comment:
@mcollina for some reason, experimental flags are being carried around as a noop [forever?], e.g. see the line above for abortcontroller. I personally disagree, as I believe the experimental flag itself should be seen as `experimental` and could be removed at any given time. with keeping those around, I believe node.js is sending the wrong signal to users.

Reply:
Ah dang, I'll make it a no-op then.

Reply:
+1 to keep it as noop. It should probably be made is a separate commit/PR, as removing the runtime warning is not the same thing as turning a flag to noop.